Ally4/backend-portfolio
# Testing of endpoints

# Description

This PR contains the tests of endpoints

## Type of change

Please check the options that are relevant.


# How should this be manually tested or reproduced?

- Clone this repo
- Checkout to `ch-tests-endpoints`
- Run the command `npm i`
- Run the command `npm test` to run the  system locally 
- Check the result in the terminal

# Checklist:

- [X] Installation in package.json
- [X] Create dummy data for admin
- [X] Create tests for admin
- [X] Create dummy data for post
- [X] Create tests for post
- [X] Create dummy data for query
- [X] Create tests for query
